### ISSUE TITLE
Pode  hosting on Azure App Service WebApp 

### ISSUE THREAD

**Issue by ehagen** [Badgerati/Pode#525]
I am starting to host Pode on Azure App Service WebApp (Windows variant == IIS) and I can unfortunately only use the powershell.exe hosting. Pwsh.exe is not yet supported on Azure App Service (Windows). Everything works except we need to disable two things:

	1. [Console]::TreatControlCAsInput = $true
	2. Write-Host
	
This means in core.ps1: 133
````powershell
# set it so ctrl-c can terminate, unless serverless
if (!$PodeContext.Server.IsServerless -and !$PodeContext.Server.IsIIS -and ($PSVersionTable.PSVersion.Major -gt 5) ) {
            [Console]::TreatControlCAsInput = $true
}
````
And in our own code or Pode  (in every runspace)

````powershell
# disable write-host in ps 5.* hosting on azure
function Global:Write-Host($Msg)
{
    # redirect output to logfile here
    #[console]::WriteLine($Msg)
}
````
I know there are better alternatives like docker but for the moment I prefer the Windows/IIS way which enables me to host on premise (IIS) and in the cloud (IIS) the same way. 

It's nice that we can host our Powershell code on the Web and don't have to resort to c# as in the past. Thanks for making this possible.

**Comment by Badgerati** [comment]
That's interesting! I would have expected `$PodeContext.Server.IsIIS` to have been set, so it would automatically skip the ControlCAsInput 🤔 would you be able to gather a list of the Environment Variables available to Pode from IIS in the Azure App Service? I'm suspecting one of the expected env vars isn't available, which would make sense if only powershell.exe is available.

If you're using IIS everywhere, I was about to suggest using `-DisableTermination` on `Start-PodeServer`, but I've just noticed that it's actually been broken since v0.15.0!! Oops! 🙈  I'll raise something for that one, because that flag should skip ControlCAsInput as it's not needed if disabled.

On `Write-Host`, I imagine this is for the "Listening" and "Terminated" messages? I could add a new `-DisableOutput` or if it's under IIS then `Write-Host`s are skipped by default.

**Comment by ehagen** [comment]
I think -DisableTermination would be sufficient if it works :)
 
Regarding Write-Host:  adding switch DisableOutput would be nice. Otherwise we can do it ourselves but now we don't forget about it.

**Comment by Badgerati** [comment]
I've fixed the `-DisableTermination` flag, and also added a new `-Quiet` switch as well which will stop all `Write-Host`/`Out-Default` output to the terminal from Pode.

**Comment by Badgerati** [comment]
Managed to spin-up an Azure Web App and have a play 😄 

I couldn't get the `[Console]::TreatControlCAsInput = $true` error to trigger, as it was successfully setting the `IsIIS` flag for me. But I did get the `Write-Host` errors!

I added a check into Pode, so if it knows it's IIS then it double checks for an Azure Web App environment variable (`WEBSITE_IIS_SITE_NAME`) - if this is found, it forces the server into `-Quiet` mode.
